modules/nginx/templates/user_tenant_nginx.conf.erb

```diff
@@ -129,21 +129,25 @@ server {
         expires -1;
     }
 
-    rewrite ^/$               /ui/index.html last;
-    rewrite ^/content         /ui/content.html last;
-    rewrite ^/dashboard       /ui/index.html last;
-    rewrite ^/discussions     /ui/index.html last;
-    rewrite ^/discussion      /ui/discussion.html last;
-    rewrite ^/folder          /ui/folder.html last;
-    rewrite ^/groups          /ui/index.html last;
-    rewrite ^/group           /ui/group.html last;
-    rewrite ^/library         /ui/index.html last;
-    rewrite ^/meetings-jitsi  /ui/index.html last;
-    rewrite ^/meeting-jitsi   /ui/meeting-jitsi.html last;
-    rewrite ^/network         /ui/index.html last;
-    rewrite ^/search          /ui/search.html last;
-    rewrite ^/signup          /ui/signup.html last;
-    rewrite ^/user            /ui/user.html last;
+    rewrite ^/$                   /ui/index.html last;
+    rewrite ^/content             /ui/content.html last;
+    rewrite ^/dashboard           /ui/index.html last;
+    rewrite ^/discussions         /ui/index.html last;
+    rewrite ^/discussion          /ui/discussion.html last;
+    rewrite ^/folder              /ui/folder.html last;
+    rewrite ^/groups              /ui/index.html last;
+    rewrite ^/group               /ui/group.html last;
+    rewrite ^/library             /ui/index.html last;
+    rewrite ^/meetings-jitsi      /ui/index.html last;
+    rewrite ^/meeting-jitsi       /ui/meeting-jitsi.html last;
+    rewrite ^/network             /ui/index.html last;
+    rewrite ^/search              /ui/search.html last;
+    rewrite ^/signup              /ui/signup.html last;
+    rewrite ^/user                /ui/user.html last;
+    rewrite ^/lti                 /ui/lti.html last;
+    rewrite ^/resetpassword       /ui/resetpassword.html last;
+    rewrite ^/resetpassword/(.*)  /ui/resetpassword.html last;
```

Comment:
This is pointless as the previous resetpassword rewrite will always be matched first.